src/components/Restoration/BusinessName.vue

```diff
@@ -1,132 +1,66 @@
 <template>
   <div id="business-name">
-    <div class="section-container" :class="{'invalid-section': invalidSection}">
-      <!-- Business Name -->
+    <pre>getNameRequestApprovedName={{ getNameRequestApprovedName }}</pre>
+    <pre>isCorrectingName={{ isCorrectingName }}</pre>
+    <!-- Display Mode -->
+    <template v-if="isNewName">
+      <NameRequestInfo />
```

Comment:
Using existing readonly component for displaying.

Below, using new component for editing.

Reply:
I changed the code around a bit but my comment is still valid: using existig component + new code for editing.